trainee_template/hyperparameters.amlg

```diff
@@ -349,13 +349,16 @@
 	(seq
 		(if (= 0 (size hyperparameterParamPaths))
 			(seq
-				(accum (assoc
-					warnings
-						(associate (concat
-							"There are no cached hyperparameters in this trainee. This operation was executed using a set of predefined default hyperparameters."
-							"Please run analyze() with your desired parameters."
-						))
-				))
+				(if (and (not autoAnalyzeEnabled ) (>= 2 (call GetNumTrainingCases)))
```

Comment:
I believe this comparator needs to be reversed (or swap the args). 

We want the warning when there are 2 or more cases. So it should be (>= num_cases 2)

Reply:
this should fix the failing test as well - I think